BDArmory/Settings/CompSettings.cs

```diff
@@ -0,0 +1,76 @@
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+using System.IO;
+using System.Collections.Generic;
+using System;
+using BDArmory.UI;
+
+namespace BDArmory.Settings
+{
+	public class CompSettings
+	{
+		// Settings overrides for AI/WM settings for competition rules compliance
+
+		static readonly string CompSettingsPath = Path.GetFullPath(Path.Combine(KSPUtil.ApplicationRootPath, "GameData/BDArmory/PluginData/Comp_settings.cfg"));
```

Comment:
Not sure what a good name for the file would be. Maybe "competition_overrides.cfg" or "custom_overrides.cfg" (some overrides also apply outside of competitions/combat)?